documents/motions/06.2021.1 - Regional Organizations.md

```diff
@@ -0,0 +1,58 @@
+# SUBMISSION OF PROPOSED MOTION
+
+**Motion number:** 6.2021.1  
+**Subject:** Regional Organizations  
+**Intent:** Procedure of acknowledgement of Regional Organizations and description of their rights and duties  
+**Submitted by:** WCA Board  
+**Date:** February 28, 2021  
+
+# Motion
+
+
+The WCA shall acknowledge Regional Organizations as Members to support the Objectives of the WCA in specific regions.
+
+1. Rights of Regional Organizations:
+   1. Regional Organizations may represent the WCA in the region that they are acknowledged for, with activities that support the Objectives of the WCA.
+   2. Regional Organizations may recognize a separate organization to act on its behalf in a subsection of the region of which it covers. This organization shall have the full rights and responsibilities of the Regional Organization.
+      1. Regional Organizations must submit to the WCA a list of the Organizations which it provides authorization to.
+      2. Regional Organizations are responsible for any and all actions of organizations that act on its behalf.
+   3. Full Regional Organizations may nominate themselves to be responsible for the payment of WCA dues for the region in which they cover.
```

Comment:
How would this work with sub-ROs? The WFC can easily implement a rule for "if competition in country X, invoice to organisation Y" but being more targeted and with exclusion rules etc. could be challenging - it'd be good if there was an "at the discretion of the WFC" or similar?

Reply:
I believe the intention is that only full RO's (so not sub RO's) can be responsible for the payment of dues. We will work on a better wording to make this more clear.

Reply:
We could add only to the start of the sentence to make it more obvious that subROs do not have this responsibility. "Only full Regional Organizations may nominate themselves to be responsible for the payment of WCA dues for the region in which they cover."

The "...organization shall have the full rights and responsibilities of the Regional Organization." part can be amended once the above wording is agreed upon, I think.

Reply:
The new wording will not allow Sub RO's this ability anymore and should cover your concerns